src/docfx/docset/Document.cs

```diff
@@ -378,7 +378,7 @@ private static bool IsValidRelativePath(string path)
 
         private (string docId, string versionIndependentId) LoadDocumentId()
         {
-            var sourcePath = PathUtility.NormalizeFile(Path.GetRelativePath(Docset.Config.DocumentId.SourceBasePath, FilePath));
+            var sourcePath = PathUtility.NormalizeFile(Path.GetRelativePath(Docset.Config.LegacyConfig.SourceBasePath, FilePath));
```

Comment:
Thx, these two configs are docset-specific, they are used for calculating source/output path per docset.  
I think if we only support single-docset for the moment, leaving them as config's immediate children makes more sense? (just remove the duplicated ones in documentId config?)

Reply:
yeah, remove the ones in documentId is better.

Reply:
these two configs are used by legacy document id logic, so if you put the source base path and site base path to our config, it will cause confusion with the routing config